src/app/components/PlayersBoard.ts

```diff
@@ -0,0 +1,102 @@
+import { Render } from '../utils/Render';
+import { flatten } from 'lodash';
+import { Player } from '../../Player';
+
+export class PlayersBoard {
+  /* returns player's board with player's name, avatar and herd */
+  renderPlayersBoard(
+    playersChosenName: string,
+    playersChosenAvatarPath: string,
+    //(when will the color in Player) playersColor: string,
+  ): HTMLElement {
+    const player = new Player(
+      playersChosenName,
+      playersChosenAvatarPath,
+    );
+    const playersBoardContainer: HTMLElement = Render.elementFactory(
+      'div',
+      {
+        className: 'board-container',
+      },
+    );
+    const playersDataPanel: HTMLElement = Render.elementFactory(
+      'div',
+      { className: 'players__data' },
+    );
+    const playerName: HTMLElement = Render.elementFactory(
+      'h3',
+      {
+        className: 'players__data__name',
+        // style: `color:${playersColor}`,
+      },
+      playersChosenName,
+    );
+    const playerAvatar: HTMLElement = Render.elementFactory('img', {
+      className: 'players__data__avatar',
+      src: playersChosenAvatarPath,
```

Comment:
should retrieve data from passed argument, so player.theAvatar (check with Player class)